invenio_rdm_migrator/streams/oauth/load.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio RDM migration OAuth load module."""
+
+from ...load.postgresql import (
+    ExistingDataTableGenerator,
+    IdentityTableGenerator,
+    PostgreSQLCopyLoad,
+)
+from .models import RemoteAccount, RemoteToken, ServerClient, ServerToken, UserIdentity
+
+
+class OAuthClientCopyLoad(PostgreSQLCopyLoad):
+    """OAuthClient loading of existing data.
+
+    It does not load the two tables with tokens, since they need rehashing.
+    """
+
+    def __init__(self, db_uri, data_dir, **kwargs):
+        """Constructor."""
+        super().__init__(
+            db_uri=db_uri,
+            table_generators=[
+                # it passes on prepare so entries wont have an effect on it
+                ExistingDataTableGenerator(tables=[RemoteAccount, UserIdentity]),
```

Comment:
Mixing existing data tg and others that require processing conflicts on the path to the files since this is decided at load level. I'm currently modifying it to be decided at tg level.